api/routes/index.js

```diff
@@ -1,74 +1,71 @@
-var express = require('express');
-var router = express.Router();
-
-var isAuthenticated = function (req, res, next) {
-	// if user is authenticated in the session, call the next() to call the next request handler 
-	// Passport adds this method to request object. A middleware is allowed to add properties to
-	// request and response objects
-	if (req.isAuthenticated()){
-		console.log('You are authrized to go to home page');
-		return next();
-	}
-	    
-	// if the user is not authenticated then redirect him to the login page
-	console.log('You are not authourized');
-	res.redirect('/');
-}
+/* 
+ * SignUp functionality to the user using Local Strategy
+ * First checks if the email or username already regestered in the DB,
+ * Creates an user in the DB with user credentials 
+ * We are hashig password using bcrypt-nodejs to make it secure
+*/
+var LocalStrategy   = require('passport-local').Strategy;

```

Comment:
this entire file should be in the routes/signup.js file. should not be overwriting the index.js file!